build/TestLocalBuild.CMD

```diff
@@ -1,42 +0,0 @@
-@echo off
```

Comment:
People are still using this script now and then. It may not be the best way to build, though. In most cases, building through the sln or any solution filter is good enough and what's in the uno.ui.build.csproj is not necessarily useful outside of the CI.